utils/src/main/kotlin/Utils.kt

```diff
@@ -198,15 +198,12 @@ fun getPathFromEnvironment(executable: String): File? {
  */
 fun fixupUserHomeProperty() {
     val userHome = System.getProperty("user.home")
-    val checkedUserHome = sequenceOf(
-        userHome,
-        Os.env["HOME"],
-        Os.env["USERPROFILE"]
-    ).find {
-        !it.isNullOrBlank() && it != "?"
+    if (userHome == null || userHome == "?") {
+        sequenceOf(
+            Os.env["HOME"],
+            Os.env["USERPROFILE"]
+        ).filterNotNull().firstOrNull()?.let { System.setProperty("user.home", it) }
```

Comment:
The `isBlank`-check is also not needed - why?

Reply:
I have added it again.